allauth/account/adapter.py

```diff
@@ -124,12 +124,14 @@ def render_mail(self, template_prefix, email, context, headers=None):
                 if ext == "txt" and not bodies:
                     # We need at least one body
                     raise
-        if "txt" in bodies:
+        if "html" in bodies and "txt" in bodies:
```

Comment:
**Question**: In case both `html` & `txt` are present, in the previous version the `txt` would go first, and `html` added as an alternative. With the changes made here this is now the other way around. Why is that needed?